docs/KubeConformance-1.13.md

```diff
@@ -0,0 +1,1538 @@
+
+A conformant Kubernetes distribution MUST support DaemonSet RollingUpdates.
+
+
+
+## [DaemonSet-Rollback](https://github.com/kubernetes/kubernetes/tree/release-1.13/test/e2e/apps/daemon_set.go#L376)
+
+A conformant Kubernetes distribution MUST support automated, minimally disruptive
+rollback of updates to a DaemonSet.
+
+
+
+## [Deployment RollingUpdate](https://github.com/kubernetes/kubernetes/tree/release-1.13/test/e2e/apps/deployment.go#L77)
+
+A conformant Kubernetes distribution MUST support the Deployment with RollingUpdate strategy.
+
+
+
+## [Deployment Recreate](https://github.com/kubernetes/kubernetes/tree/release-1.13/test/e2e/apps/deployment.go#L84)
+
+A conformant Kubernetes distribution MUST support the Deployment with Recreate strategy.
+
+
+
+## [Deployment RevisionHistoryLimit](https://github.com/kubernetes/kubernetes/tree/release-1.13/test/e2e/apps/deployment.go#L92)
+
+A conformant Kubernetes distribution MUST clean up Deployment's ReplicaSets based on
+the Deployment's `.spec.revisionHistoryLimit`.
+
+
+
+## [Deployment Rollover](https://github.com/kubernetes/kubernetes/tree/release-1.13/test/e2e/apps/deployment.go#L101)
+
+A conformant Kubernetes distribution MUST support Deployment rollover,
+i.e. allow arbitrary number of changes to desired state during rolling update
+before the rollout finishes.
+
+
+
+## [Deployment Proportional Scaling](https://github.com/kubernetes/kubernetes/tree/release-1.13/test/e2e/apps/deployment.go#L119)
+
+A conformant Kubernetes distribution MUST support Deployment
+proportional scaling, i.e. proportionally scale a Deployment's ReplicaSets
+when a Deployment is scaled.
+
+
+
+## [Replication Controller, run basic image](https://github.com/kubernetes/kubernetes/tree/release-1.13/test/e2e/apps/rc.go#L47)
+
+### Release v1.9
+Replication Controller MUST create a Pod with Basic Image and MUST run the service with the provided image. Image MUST be tested by dialing into the service listening through TCP, UDP and HTTP.
+
+
+
+## [Replication Controller, adopt matching pods](https://github.com/kubernetes/kubernetes/tree/release-1.13/test/e2e/apps/rc.go#L68)
+
+### Release v1.13
+An ownerless Pod is created, then a Replication Controller (RC) is created whose label selector will match the Pod. The RC MUST either adopt the Pod or delete and replace it with a new Pod
+
+
+
+## [Replication Controller, release pods](https://github.com/kubernetes/kubernetes/tree/release-1.13/test/e2e/apps/rc.go#L77)
+
+### Release v1.13
+A Replication Controller (RC) is created, and its Pods are created. When the labels on one of the Pods change to no longer match the RC's label selector, the RC MUST release the Pod and update the Pod's owner references.
+
+
+
+## [Replica Set, run basic image](https://github.com/kubernetes/kubernetes/tree/release-1.13/test/e2e/apps/replica_set.go#L90)
+
+### Release v1.9
```

Comment:
Why does this say v1.9? Does it mean it was added in 1.9 or?

Reply:
The version indicates the Kubernetes release in which the test was added.

Reply:
ok